src/main/webapp/resources/js/components/AnalysesTable/AnalysesTable.jsx

```diff
@@ -104,48 +95,52 @@ export function AnalysesTable() {
       {
         title: getI18N("analysis.duration"),
         key: "duration",
+        width: 150,
         dataIndex: "duration",
         render(timestamp) {
           return getHumanizedDuration({ date: timestamp });
         }
+      },
+      {
+        title: "",
+        key: "download",
+        fixed: "right",
+        render(text, record) {
+          return (
+            <Button
+              shape="circle-outline"
+              disabled={record.state.value !== "COMPLETED"}
+              href={`${window.TL.BASE_URL}ajax/analyses/download/${record.id}`}
+              download
+              icon="download"
+            />
+          );
+        }
       }
     ];
 
-    if (ADMIN) {
+    if (CAN_MANAGE) {
       columns.push({
         title: "",
-        key: "actions",
+        key: "delete",
         fixed: "right",
         render(text, record) {
           return record.modifiable ? (
-            <div
-              style={{
-                display: "flex",
-                justifyContent: "space-between",
-                alignItems: "center"
-              }}
+            <Popconfirm
+              placement={"top"}
+              An
+              title={"Delete this analysis?"}
```

Comment:
Internationalization

Reply:
This button is removed in the next merge request, so don't worry about this now.